Mesh_2/include/CGAL/Mesh_2/Refine_edges_with_clusters.h

```diff
@@ -274,8 +274,8 @@ class Refine_edges_base_with_clusters :
 
         typename Geom_traits::Vector_2 v = vector(a,m);
         v = scaled_vector(
-            v, CGAL_NTS sqrt(CGAL::to_double(c.minimum_squared_length /
-                                             squared_distance(a, b))));
+            v, CGAL_NTS sqrt(CGAL_NTS to_double(c.minimum_squared_length /
```

Comment:
I may be wrong but the `CGAL_NTS` has become useless, as it is an empty `#define`

Reply:
That's what I thought too, but since there seemed to be an error in the checks this seemed the most likely cause.

Reply:
I looks more consistent now too. :)